app-misc/mosquitto/mosquitto-1.4.15.ebuild

```diff
@@ -0,0 +1,102 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+PYTHON_COMPAT=( python2_7 )
+
+inherit systemd user toolchain-funcs python-any-r1
+
+DESCRIPTION="An Open Source MQTT v3 Broker"
+HOMEPAGE="http://mosquitto.org/"
+SRC_URI="http://mosquitto.org/files/source/${P}.tar.gz"
+
+LICENSE="EPL-1.0"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~x86"
+IUSE="bridge examples +persistence +srv ssl tcpd websockets systemd"
+
+RDEPEND="tcpd? ( sys-apps/tcp-wrappers )
+	ssl? ( dev-libs/openssl:0= )"
+DEPEND="${RDEPEND}
+	${PYTHON_DEPS}
+	srv? ( net-dns/c-ares )
+	websockets? ( net-libs/libwebsockets )"
+
+makeopts() {
+		"CC=$(tc-getCC)"
+		"LIB_SUFFIX=${LIBDIR:3}"
+		"WITH_BRIDGE=$(usex bridge)"
+		"WITH_PERSISTENCE=$(usex persistence)"
+		"WITH_SRV=$(usex srv)"
+		"WITH_TLS=$(usex ssl)"
+		"WITH_WEBSOCKETS=$(usex websockets)"
+		"WITH_WRAP=$(usex tcpd)"
+}
+
+pkg_setup() {
+	enewgroup mosquitto
+	enewuser mosquitto -1 -1 -1 mosquitto
+}
+
+src_prepare() {
+	eapply "${FILESDIR}/${PN}-1.4.10-conditional-tests.patch"
+	if use persistence; then
+		sed -i -e "s:^#autosave_interval:autosave_interval:" \
+			-e "s:^#persistence false$:persistence true:" \
+			-e "s:^#persistence_file:persistence_file:" \
+			-e "s:^#persistence_location$:persistence_location /var/lib/mosquitto/:" \
+			mosquitto.conf || die
+	fi
+
+	# Remove prestripping
+	sed -i -e 's/-s --strip-program=${CROSS_COMPILE}${STRIP}//'\
+		client/Makefile lib/cpp/Makefile src/Makefile lib/Makefile || die
+
+	python_setup
+	python_fix_shebang test
+	eapply_user
+}
+
+src_configure() {
+	econf --lib=$(get_libdir)
+}
+
+src_compile() {
+	emake $(makeopts)
+}
+
+src_test() {
+	emake $(makeopts) test
+}
+
+src_install() {
+	emake $(makeopts) DESTDIR="${D}" prefix="${EPREFIX}/usr" install
+	keepdir /var/lib/mosquitto
+	fowners mosquitto:mosquitto /var/lib/mosquitto
+	dodoc readme.md CONTRIBUTING.md ChangeLog.txt
+	doinitd "${FILESDIR}"/mosquitto
+	insinto /etc/mosquitto
+	doins mosquitto.conf
+
+	if use systemd; then
```

Comment:
This is invalid use of USE=systemd.

Reply:
How so? That is how `examples` is being used. Where are the rules against this?

Reply:
```
$ quse -D systemd
 global:systemd: Enable use of systemd-specific libraries and features like socket activation or session tracking
```

https://wiki.gentoo.org/wiki/Project:Systemd/Ebuild_policy#General_guidelines

Reply:
"Please note that unit files are still installed unconditionally to the USE flag."

I see this now, thank you.